usr/tgtd.c

```diff
@@ -988,6 +994,7 @@ static int new_stord(const _ha_request *reqp,
 	}
 
 	pthread_mutex_lock(&ha_rest_mutex);
+	strcat(hyc_argv[1], tgt_idx);
```

Comment:
- prefer to use strncat
- how is `json_t* tgt_index` converted to `const char*`.
  `char *strcat(char *dest, const char *src);`